compiler/rustc_span/src/symbol.rs

```diff
@@ -881,6 +881,7 @@ symbols! {
         expf16,
         expf32,
         expf64,
+        explicit_extern_abis,
```

Comment:
Add a section if you would to the unstable book.  See:

https://rustc-dev-guide.rust-lang.org/implementing_new_features.html#stability-in-code